### ISSUE TITLE
Introspection with standard values does not work

### ISSUE THREAD

**Issue by ZirgVoice** [GraphQLSwift/GraphQL#152]
When using the default value, introspection does not work.
For the `ID` scalar, I get the error `Serialize expected type ID but got Map`.
For `Bool`, the error is `Cannot convert value to AST: 0`.
For `Int`,` Cannot convert value to AST: 1`.
This is what I managed to check. `Bool` definitely worked before update `2.10.1 - Paramaribo`

**Comment by NeedleInAJayStack** [comment]
Hey, this is interesting. Are you declaring the type as Swift's `Bool` or `GraphQLBool` like done in the test here? https://github.com/GraphQLSwift/GraphQL/blob/main/Tests/GraphQLTests/TypeTests/IntrospectionTests.swift#L40

**Comment by ZirgVoice** [comment]
No, I do it like this
```swift
class Schema: PartialSchema<GraphQLResolver, Context> {

    // MARK: - Query
    @FieldDefinitions
    override var query: Fields {
        Field("raceSession", at: GraphQLResolver.raceSession) {
            Argument("restart", at: \.restart).defaultValue(false)
        }
    }
}
```

**Comment by NeedleInAJayStack** [comment]
I tried to recreate this issue in Graphiti on [this branch](https://github.com/NeedleInAJayStack/Graphiti/commits/fix/issue-152/), see [this commit](https://github.com/NeedleInAJayStack/Graphiti/commit/540c4fce527654ef71392da6b1b824127d3266c1), but wasn't able to. The test runs successfully.

Could you edit this test or create a new one reproducing the errors you are seeing?

**Comment by ZirgVoice** [comment]
I noticed that you're calling the query in the tests, but the problem is not with them. The error is in the response to the introspection query. Here’s an example of the call
This is a `POST` request
```
{"query":"\n    query IntrospectionQuery {\n      __schema {\n        \n        queryType { name }\n        mutationType { name }\n        subscriptionType { name }\n        types {\n          ...FullType\n        }\n        directives {\n          name\n          description\n          \n          locations\n          args {\n            ...InputValue\n          }\n        }\n      }\n    }\n\n    fragment FullType on __Type {\n      kind\n      name\n      description\n      \n      fields(includeDeprecated: true) {\n        name\n        description\n        args {\n          ...InputValue\n        }\n        type {\n          ...TypeRef\n        }\n        isDeprecated\n        deprecationReason\n      }\n      inputFields {\n        ...InputValue\n      }\n      interfaces {\n        ...TypeRef\n      }\n      enumValues(includeDeprecated: true) {\n        name\n        description\n        isDeprecated\n        deprecationReason\n      }\n      possibleTypes {\n        ...TypeRef\n      }\n    }\n\n    fragment InputValue on __InputValue {\n      name\n      description\n      type { ...TypeRef }\n      defaultValue\n      \n      \n    }\n\n    fragment TypeRef on __Type {\n      kind\n      name\n      ofType {\n        kind\n        name\n        ofType {\n          kind\n          name\n          ofType {\n            kind\n            name\n            ofType {\n              kind\n              name\n              ofType {\n                kind\n                name\n                ofType {\n                  kind\n                  name\n                  ofType {\n                    kind\n                    name\n                  }\n                }\n              }\n            }\n          }\n        }\n      }\n    }\n  ","operationName":"IntrospectionQuery"}
```

At the end of the response to this call, there is an error
```
                }
            ]
        }
    },
    "errors": [
        {
            "message": "Cannot convert value to AST: 0",
            "path": []
        }
    ]
}
```

**Comment by ZirgVoice** [comment]
Here in tests I reproduced the error: https://github.com/GraphQLSwift/Graphiti/compare/main...ZirgVoice:Graphiti:issus-152

**Comment by NeedleInAJayStack** [comment]
> The error is in the response to the introspection query

Oh duh, I was able to recreate in my tests by making the introspection query.

Alright, I believe this is related to the serialization of the standard GraphQL Scalar types. I've got a fix here that resolves my reproducer (I haven't added tests yet): https://github.com/NeedleInAJayStack/GraphQL/commits/fix/issue-152/

**Comment by NeedleInAJayStack** [comment]
This should now be fixed in [2.10.2](https://github.com/GraphQLSwift/GraphQL/releases/tag/2.10.2)
